src/component.cpp

```diff
@@ -37,8 +37,8 @@ struct Component::ComponentImpl
 
 std::vector<VariablePtr>::iterator Component::ComponentImpl::findVariable(const std::string &name)
 {
-    return std::find_if(mVariables.begin(), mVariables.end(),
-                        [=](const VariablePtr& v) -> bool {return v->getName() == name;});
+    return std::find_if (mVariables.begin(), mVariables.end(),
```

Comment:
Fair enough- I didn't see an instance of find_if in the Google style guide but did see it on [cplusplus.com](http://www.cplusplus.com/reference/algorithm/find_if/) with spacing. But that makes sense and my Stroustrup book has it without the spacing.